### ISSUE TITLE
E101 should ignore multiline strings

### ISSUE THREAD

**Issue by konstin** [astral-sh/ruff#8443]
In the code below, the whitespace ahead of a and b contains tabs and spaces:
```python
from pathlib import Path


def b():
    Path("my_data.txt").write_text("""
    	a
    		b
    """)

```
This raises "E101 Indentation contains mixed spaces and tabs", even though this is not indentation but the contents of a multiline string where the leading indentation is user data and not python code. We should ignore multiline strings for E101.

**Comment by charliermarsh** [comment]
I believe this is a dupe of https://github.com/astral-sh/ruff/issues/7906 where I was asking for a situation in which this would be used in practice.
